source/tyr/migrations/versions/6f2635ce00b4_add_max_taxi_duration_to_pt.py

```diff
@@ -0,0 +1,22 @@
+"""add max_taxi_duration_to_pt
+
+Revision ID: 6f2635ce00b4
+Revises: 858b24967ff2
+Create Date: 2019-12-04 12:12:42.130204
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '6f2635ce00b4'
+down_revision = '858b24967ff2'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('instance', sa.Column('max_taxi_duration_to_pt', sa.Integer(), nullable=False))
```

Comment:
Does this work? The column can't be null but it has no default value?

Reply:
you're right!